### PR TITLE
Add carto-vl guide

### PR DESCRIPTION
**Pull request by IagoLast**
#240
---

### CODE REVIEW

File: docs/guides/00-integration-carto-vl.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+## Adding a Category Widget
+
+Adding a Category Widget is as simple as including `<as-category-widget>` tag within our `as-sidebar` element, setting the options you prefer.
+
+ ```html
+<aside class="as-sidebar as-sidebar--right">
+  <as-category-widget
+    id="cities-widget"
+    heading="Populated places"
+    description="Sum of the population grouped by country">
```
Comment (jesusbotella) [inline docs/guides/00-integration-carto-vl.md]: You need to change this description, as it was the previously one I used in my widget before changing widget data.

File: docs/guides/00-integration-carto-vl.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        name: feature.name,
+        value: feature.pop_max,
+      }))
+      .sort((a, b) => b.value - a.value);
+  };
+```
+
+### Reacting to data changes
+
+> Waiting for a new API in CARTO-VL
```
Comment (ivanmalagon) [inline docs/guides/00-integration-carto-vl.md]: Is this related to filtering or to moving the map?

Comment (IagoLast) [reply docs/guides/00-integration-carto-vl.md]: non implemented feature in CARTO-VL, actually @Jesus89 had a similar problem at the same time
